openghg/standardise/_standardise.py

```diff
@@ -875,3 +875,99 @@ def standardise_from_binary_data(
             binary_data=binary_data, metadata=metadata, file_metadata=file_metadata, **kwargs
         )
     return result
+
+
+def standardise_flux_timeseries(
+    filepath: Union[str, Path],
+    species: str,
+    source: str,
+    domain: Optional[str] = None,
+    region: str = "UK",
+    source_format: str = "crf",
+    database: Optional[str] = None,
+    database_version: Optional[str] = None,
+    model: Optional[str] = None,
+    store: Optional[str] = None,
+    if_exists: str = "auto",
+    save_current: str = "auto",
+    overwrite: bool = False,
+    force: bool = False,
+    compressor: Optional[Any] = None,
+    filters: Optional[Any] = None,
+    period: Optional[Union[str, tuple]] = None,
+    continuous: Optional[bool] = None,
+    optional_metadata: Optional[Dict] = None,
+) -> Dict:
+    """Process one dimension timeseries file
+
+    Args:
+        filepath: Path of flux timeseries file
+        species: Species name
+        domain: Geographic domain, default is 'None'. Instead region is used to identify area
```

Comment:
```suggestion
        domain: If flux is related to pre-existing domain (e.g. "EUROPE") with defined latitude-longitude bounds this can be used to flag that. Otherwise, use `region` input to describe the name of a region (e.g. "UK").
```
As per the suggestion on argument order, the arguments should be reordered in the doc-string in the same way.